pkg/server/handler/v1alpha1/helper.go

```diff
@@ -43,6 +43,43 @@ func getLogFolder(tenant, project, workflow, workflowrun string) (string, error)
 	return strings.Join([]string{cycloneHome, tenant, project, workflow, workflowrun, logsFolderName}, string(os.PathSeparator)), nil
 }
 
+// deleteCollections deletes collections in the sub paths of a tenant in the pvc, collections including:
+// - logs
+// - artifacts (WIP)
+// subpaths could be:
+// - project
+// - workflow
+// - workflowrun
+func deleteCollections(tenant string, subpaths ...string) error {
+	if tenant == "" {
+		return fmt.Errorf("tenant can not be empty")
+	}
+
+	// get collection folder
+	folder, err := getCollectionFolder(tenant, subpaths...)
+	if err != nil {
+		log.Errorf("delete collections of tenant %s subpaths %s, err:%v", tenant, subpaths, err)
+		return err
+	}
+
+	// remove collection folder
+	if err := os.RemoveAll(folder); err != nil {
+		log.Errorf("remove folder %s error:%v", folder, err)
+		return err
+	}
+
+	return nil
+}
+
+// getProjectFolder gets the folder path for the project.
+func getCollectionFolder(tenant string, subpaths ...string) (string, error) {
+	ret := strings.Join([]string{cycloneHome, tenant}, string(os.PathSeparator))
```

Comment:
paths := []string{cycloneHome, tenant}
paths = append(paths, subpaths...)
return strings.Join(paths, string(os.PathSeparator))

Reply:
no need to return error

Reply:
Fixed